### PR TITLE
Mitchaudio  mini preview responds to volume changes

### PR DESCRIPTION
**Pull request by mikemitchel**

### CODE REVIEW

File: client/createStory/createStory.js [hunk 1]
```diff
@@ -446,13 +447,16 @@ angular.module('storyBoard.createStory', [])
...
+    var currentFrame = {
+      playerDiv: frameDivId,
+      videoId: videoId,
+      start: frameStartTime,
+      end: frameEndTime,
+      volume: frameVolume,
+      playerName: framePlayerName
+    };
+
+    $scope.previewAudioVideoFrame(currentFrame);
```
Comment (jd009) [inline client/createStory/createStory.js]: If this is only called internally, we don't need to expose it on the dollar-sign scope.  We could just declare the function such that it is private to the controller:
```
var previewAudioVideoFrame = function(currentFrame){
}
```
In doing so, you communicate to everyone else that this function can't be called from the HTML.

File: client/createStory/createStory.js [hunk 2]
```diff
@@ -529,12 +534,53 @@ angular.module('storyBoard.createStory', [])
...
+              break;
+              case 'frame2':
+
+                        $scope.showSpinner2 = false;
+
+              break;
+              case 'frame3':
+
+                        $scope.showSpinner3 = false;
+
```
Comment (jd009) [inline client/createStory/createStory.js]: There's a lot of extra spaces within this switch statement.  Also, the indentation got massively larger than our standard 2 spaces.

File: client/createStory/createStory.js [hunk 3]
```diff
@@ -551,9 +597,9 @@ angular.module('storyBoard.createStory', [])
         $scope.addFrame3ImagePreview = true;
         break;
     }
-  }
+  };
 
-  $scope.previewTextToSpeech = function(frameId){
+  $scope.previewTextToSpeechFrame = function(frameId){
```
Comment (jd009) [inline client/createStory/createStory.js]: If we change the name of a dollar-sign scope function, we need to make a corresponding change in the HTML where it is called.

**Comment by mikemitchel** [comment]
fixed - spinner logic in next pull request

**Pull request merged**